play/src/pages/PregameCountdown.tsx

```diff
@@ -22,33 +23,34 @@ const TypographyStyled = styled(Typography)(({ theme }) => ({
 }));
 
 interface StartPhase2Props {
-  handlePregameTimerFinished: () => void;
+  setIsPregameCountdown: (isPregameCountdown: boolean) => void;
 }
 
 export default function StartPhase2({
-  handlePregameTimerFinished,
+  setIsPregameCountdown,
 }: StartPhase2Props) {
   const theme = useTheme();
   const { t } = useTranslation();
 
   return (
     <BackgroundContainerStyled>
       <StackContainer spacing={5}>
-        <RadialTimer
-          inputColors={[
-            `${theme.palette.primary.countdownColor}, 0.3)`,
-            `${theme.palette.primary.countdownColor}, 0.4)`,
-            `${theme.palette.primary.countdownColor}, 0.5)`,
-            `${theme.palette.primary.countdownColor}, 0.6)`,
-            `${theme.palette.primary.countdownColor}, 0.7)`,
-            `${theme.palette.primary.countdownColor}, 0.8)`,
-            `${theme.palette.primary.countdownColor}, 0.9)`,
-            `${theme.palette.primary.countdownColor}, 1)`,
-          ]}
-          radius={110}
-          timerStartInSeconds={3}
-          handlePregameTimerFinished={handlePregameTimerFinished}
-        />
+          <RadialTimer
+            mode={TimerMode.COUNTDOWN}
+            inputColors={[
+              `${theme.palette.primary.countdownColor}, 0.3)`,
+              `${theme.palette.primary.countdownColor}, 0.4)`,
+              `${theme.palette.primary.countdownColor}, 0.5)`,
+              `${theme.palette.primary.countdownColor}, 0.6)`,
+              `${theme.palette.primary.countdownColor}, 0.7)`,
+              `${theme.palette.primary.countdownColor}, 0.8)`,
+              `${theme.palette.primary.countdownColor}, 0.9)`,
+              `${theme.palette.primary.countdownColor}, 1)`,
+            ]}
```

Comment:
Is this the same colors from Lobby? If yes, maybe we can extract it in theme class as well or somebody else so it can be referenced in both places instead of being duplicated

Reply:
nice, added to `Theme` under `radialTimerArray`